src/ds/test/map_test.cpp

```diff
@@ -18,8 +22,9 @@ struct CollisionHash
   }
 };
 
-using K = uint64_t;
-using V = uint64_t;
+using K = kv::serialisers::SerialisedEntry;
+using V = kv::serialisers::SerialisedEntry;
+constexpr static size_t max_key_value_size = 128;
```

Comment:
The `map_test` now uses the underlying type for `K` and `V`, as used by CCF, so that the unit tests are closer to the actual framework implementation and that we can exercise sizes of different keys/values more conveniently.